purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -245,14 +245,28 @@ fun Purchases.identifyWith(
  * @param [onError] Will be called after the call has completed with an error.
  */
 @Suppress("unused")
-fun Purchases.logInWith(
+internal fun Purchases.logInWith(
     appUserID: String,
     onError: ErrorFunction = ON_ERROR_STUB,
     onSuccess: ReceiveLogInSuccessFunction
 ) {
     logIn(appUserID, logInSuccessListener(onSuccess, onError))
 }
 
+/**
+ * This function will change the current appUserID.
+ * Typically this would be used after a log out to identify a new user without calling configure
```

Comment:
I think this docs are for login and not for logout?

Reply:
🤦